hydra-haskell/src/main/haskell/Hydra/Impl/Haskell/Sources/Ext/Shacl/Model.hs

```diff
@@ -152,7 +225,23 @@ shaclModel = Graph shaclModelName elements (const True) hydraCoreName
           field "minCount" $
             doc ("The minimum cardinality. Node shapes cannot have any value for sh:minCount. " ++
                  "See https://www.w3.org/TR/shacl/#MinCountConstraintComponent") $
-            optional int32],
+            optional int32,
```

Comment:
We/I actually should have used `bigint` (which corresponds to `xsd:integer`) here instead of `int32`. The latter is certainly sufficient, but the former is more aligned with the spec.